MonoGame.Framework/Game.cs

```diff
@@ -612,10 +612,10 @@ public void Tick()
                 OnExiting(this, exitingEventArgs);
 
                 if (!exitingEventArgs.Cancel)
-                {
+                {
+                    UnloadContent();

```

Comment:
Do we have any unit tests that cover this change?

Reply:
@tomspilman we do not. Only way I can think we could unit test this would be to add events for Loading/Unloading contents, then we can make sure which order things are fired. Is there some other way?